lambda/templates/av_lambda.json.tpl

```diff
@@ -69,7 +69,8 @@
       "Sid": "DecryptEnvVar",
       "Effect": "Allow",
       "Action": [
-        "kms:Decrypt"
+        "kms:Decrypt",
+        "kms:GenerateDataKey"
       ],
```

Comment:
The GenerateDataKey permission is needed to allow the lambda to send messages to an encrypted SQS queue. There's more details in [the documentation](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-key-management.html#send-to-encrypted-queue)